cmd/crc/cmd/podman_env.go

```diff
@@ -32,24 +33,16 @@ func runPodmanEnv() error {
 	if err := checkIfMachineMissing(client); err != nil {
 		return err
 	}
-
-	connectionDetails, err := client.ConnectionDetails()
-	if err != nil {
+	p := podman.UsePodman()
+	if err := p.MakeRootlessSystemConnectionDefault(); err != nil {
```

Comment:
I don't expect `podman-env` to change my system configuration, I only expect it to set some environment variable and that's it. Feels like we are doing too much (setting both env vars and calling `podman system config` when only one of these is enough to achieve what we need).